src/app/app.component.scss

```diff
@@ -1,2 +1,12 @@
 @import "./../sass/vars.scss";
-@import "./../sass/mixins.scss";
\ No newline at end of file
+@import "./../sass/mixins.scss";
+
+fly-nav-menu {
```

Comment:
Let's find a shade for the nav bar that looks good but doesn't match the background of any other sections